.github/workflows/continous-integration.yaml

```diff
@@ -0,0 +1,139 @@
+name: Continous Integration
+
+# This workflow is triggered on pushes to the repository.
+on: [push, pull_request]
+
+env:
+  CI: true
+  NODE_OPTIONS: --max-old-space-size=4096
+
+jobs:
+  test-linux:
+    name: Test (Linux)
+    defaults:
+      run:
+        shell: 'script -qec "bash {0}"'
```

Comment:
Added this as a workaround to enable tty (see https://github.com/actions/runner/issues/241#issuecomment-745902718).

Unfortunately this means a duplicated, separate job for Linux only as [YAML anchors aren't supported yet](https://github.community/t/support-for-yaml-anchors/16128/47).